addons/delivery/views/report_deliveryslip.xml

```diff
@@ -1,19 +1,19 @@
 <odoo>
     <template id="report_delivery_document2" inherit_id="stock.report_delivery_document">
         <xpath expr="//div[@name='div_sched_date']" position="after">
-            <div t-if="o.picking_type_id.code == 'outgoing' and o.carrier_id" class="col-auto">
+            <div t-if="o.picking_type_id.code == 'outgoing' and o.carrier_id" class="col-auto col-3 mw-100 mb-2">
                 <strong>Carrier:</strong>
-                <p t-field="o.carrier_id"/>
+                <p class="m-0" t-field="o.carrier_id"/>
```

Comment:
Could you please put `t-field` before `class` like this everywhere?
```suggestion
                <p t-field="o.carrier_id" class="m-0"/>
```

Reply:
OK, I can change this, but it does not follow the standard of other reports.

- https://github.com/odoo/odoo/blob/16.0/addons/sale/report/ir_actions_report_templates.xml#L46
- https://github.com/odoo/odoo/blob/16.0/addons/account/views/report_invoice.xml#L69

Reply:
Do you want me to change it?

Reply:
Done